apps/site/assets/css/_event-details.scss

```diff
@@ -80,92 +88,79 @@ $separator-border: solid 1px $gray-lightest;
 }
 
 .m-event__agenda {
-  // top level and second level topics
-  .agenda-topic,
-  .agenda-topics ol > li {
-    border-top: $separator-border;
-    padding: $base-spacing * .75 0 $base-spacing;
+  p {
+    line-height: $headings-line-height;
+    margin-bottom: 0;
   }
 
-  // top-level topic
-  .agenda-topic {
-    @include agenda-bullet($counter: topicCounter, $counter-type: upper-alpha);
```

Comment:
I think we've lost the sub-topic counter reset. The numbered subtopics should reset to `1` after each topic. This preview (may be outdated) shows this is in error:

![image](https://user-images.githubusercontent.com/688435/140335254-25206277-d6ab-46dc-ab5a-ac3a6cd8d89f.png)

Reply:
This is intentional.  It matches the mocks:  https://zeroheight.com/2fedee66c/p/42cbbf-event-pages/i/31027605

Reply:
This does not seem correct, even in the mocks. Can you please confirm with design? It makes no sense to me for sub-topics to have a cross-topic number progression... it might be a design bug.

Reply:
Yinka's not available for a few days, but in past conversations, we've discussed the numbering.  That day in the summer when we were meeting as a full group to talk through the tech challenges of this design, I said I was worried we wouldn't be able to implement that style of numbering -- turned out to be an easy solution, so no worries there.  But Yinka didn't say it was a mistake.

Viewed as a full agenda, it's really easy to tell someone to look at agenda item 5.  Little faster than specifying C3.

Reply:
OK. Very awkward from a semantic POV IMO, but fine. You can go further and make CSS rules for applying "C1, C2" as part of the numbering scheme, I believe, just as an FYI.

https://clickhelp.com/clickhelp-technical-writing-blog/advanced-autonumbering-techniques-with-css/

```
1 item
   1.a
   1.b
2 item
   2.a
```